### ISSUE TITLE
Will not work if node is installed with NVM

### ISSUE THREAD

**Issue by phuchle** [5-gwoap/slng-node#2]
To replicate: uninstall any global node.js
Install node 8.9 with NVM and run `npm i -g slng`
Output:
```
$ slng <string>
/usr/bin/env: ‘node --harmony’: No such file or directory
```

Possible solution here: https://github.com/creationix/nvm/issues/786

**Comment by rainrivas** [comment]
@phuchle ~care to checkout the bugfix branch and run npm install -g > npm test and let me know if that solves your issue?~

---

**EDIT:** I have published a BETA build of 0.1.3 so if you install the package @beta, i.e. `npm install -g slng@beta` you'll have the package in question and can confirm that it works

**Comment by rainrivas** [comment]
Fixed in v0.1.3. Travis output below:
```
Updating nvm
nvm.install
2.30s$ nvm install lts/*
Downloading and installing node v8.9.3...
Downloading https://nodejs.org/dist/v8.9.3/node-v8.9.3-linux-x64.tar.xz...
######################################################################## 100.0%
Computing checksum with sha256sum
Checksums matched!
Now using node v8.9.3 (npm v5.5.1)
$ node --version
v8.9.3
$ npm --version
5.5.1
$ nvm --version
0.33.8
before_install
2.21s$ npm i -g
/home/travis/.nvm/versions/node/v8.9.3/bin/slng -> /home/travis/.nvm/versions/node/v8.9.3/lib/node_modules/slng/index.js
+ slng@0.1.3
added 54 packages in 1.745s
install.npm
0.87s$ npm install 
up to date in 0.399s
0.74s$ npm ls
slng@0.1.3 /home/travis/build/5-gwoap/slng-node
├── commander@2.12.2
└─┬ request@2.83.0
  ├── aws-sign2@0.7.0
  ├── aws4@1.6.0
  ├── caseless@0.12.0
  ├─┬ combined-stream@1.0.5
  │ └── delayed-stream@1.0.0
  ├── extend@3.0.1
  ├── forever-agent@0.6.1
  ├─┬ form-data@2.3.1
  │ ├── asynckit@0.4.0
  │ ├── combined-stream@1.0.5 deduped
  │ └── mime-types@2.1.17 deduped
  ├─┬ har-validator@5.0.3
  │ ├─┬ ajv@5.5.2
  │ │ ├── co@4.6.0
  │ │ ├── fast-deep-equal@1.0.0
  │ │ ├── fast-json-stable-stringify@2.0.0
  │ │ └── json-schema-traverse@0.3.1
  │ └── har-schema@2.0.0
  ├─┬ hawk@6.0.2
  │ ├─┬ boom@4.3.1
  │ │ └── hoek@4.2.0 deduped
  │ ├─┬ cryptiles@3.1.2
  │ │ └─┬ boom@5.2.0
  │ │   └── hoek@4.2.0 deduped
  │ ├── hoek@4.2.0
  │ └─┬ sntp@2.1.0
  │   └── hoek@4.2.0 deduped
  ├─┬ http-signature@1.2.0
  │ ├── assert-plus@1.0.0
  │ ├─┬ jsprim@1.4.1
  │ │ ├── assert-plus@1.0.0 deduped
  │ │ ├── extsprintf@1.3.0
  │ │ ├── json-schema@0.2.3
  │ │ └─┬ verror@1.10.0
  │ │   ├── assert-plus@1.0.0 deduped
  │ │   ├── core-util-is@1.0.2
  │ │   └── extsprintf@1.3.0 deduped
  │ └─┬ sshpk@1.13.1
  │   ├── asn1@0.2.3
  │   ├── assert-plus@1.0.0 deduped
  │   ├─┬ bcrypt-pbkdf@1.0.1
  │   │ └── tweetnacl@0.14.5 deduped
  │   ├─┬ dashdash@1.14.1
  │   │ └── assert-plus@1.0.0 deduped
  │   ├─┬ ecc-jsbn@0.1.1
  │   │ └── jsbn@0.1.1 deduped
  │   ├─┬ getpass@0.1.7
  │   │ └── assert-plus@1.0.0 deduped
  │   ├── jsbn@0.1.1
  │   └── tweetnacl@0.14.5
  ├── is-typedarray@1.0.0
  ├── isstream@0.1.2
  ├── json-stringify-safe@5.0.1
  ├─┬ mime-types@2.1.17
  │ └── mime-db@1.30.0
  ├── oauth-sign@0.8.2
  ├── performance-now@2.1.0
  ├── qs@6.5.1
  ├── safe-buffer@5.1.1
  ├── stringstream@0.0.5
  ├─┬ tough-cookie@2.3.3
  │ └── punycode@1.4.1
  ├─┬ tunnel-agent@0.6.0
  │ └── safe-buffer@5.1.1 deduped
  └── uuid@3.1.0
0.52s$ npm test
> slng@0.1.3 test /home/travis/build/5-gwoap/slng-node
> slng gucci
=======================
Good, chill, cool, awesome...
Comes from Gucci as in the fashion label... because Gucci is rich and ballin' therefore he is good, cool, and just pimpin'
=======================
A fashion comany started by Guccio Gucci in the late 1800s in Italy. It is now an international fashion company and is the benchmark for high-end designer clothes. Their clothes are now designed by Tom Ford. They are the makers of many lines including fragrance, jewelry, and watches. Is most commonly known for their 'double G' logo. Recently, because gucci clothing is of such high quality a bunch of ghetto idiots decided to make a term out of this prestigious name meaning cool or flashy.
=======================
a word that means good, great, fine,awesome and ect.
```
